db_stress_tool/no_batched_ops_stress.cc

```diff
@@ -333,12 +376,20 @@ class NonBatchedOpsStressTest : public StressTest {
   }
 
   void ContinuouslyVerifyDb(ThreadState* thread) const override {
-    if (!cmp_db_) {
+    // Currently this method gets called even when
+    // FLAGS_continuous_verification_interval == 0 as long as
+    // FLAGS_verify_db_one_in > 0. Previously, this was not causing a problem in
+    // the crash tests since test_secondary was always equal to 0, and thus we
+    // returned early from this method. When test_secondary is set and we have a
+    // secondary_db_, the crash test fails during this iterator scan. The stack
+    // trace mentions BlobReader/BlobSource but it may not necessarily be
+    // related to BlobDB
+    if (!secondary_db_ || !FLAGS_continuous_verification_interval) {
```

Comment:
Does this mean verify_db_one_in > 0 is ignored unless test_secondary is set?

Reply:
Okay let me update the comment since now that I read it, it is confusing. By "currently this method gets called," I really should have said "currently (before this PR which makes a change), this is what was happening, and we do _not_ want this to occur.

We do not want `verify_db_one_in > 0` to matter if `FLAGS_continuous_verification_interval` is 0